targets/mbedos5/tools/generate_pins.py

```diff
@@ -0,0 +1,224 @@
+#!/usr/bin/env python
+
+# Copyright (c) 2016 ARM Limited
+
```

Comment:
This is not exactly the header found in other files and suggested in previous review. The missing `#` seems like a minor technicality, but most probably we will have an automated license checker set up soon, and it will choke on such differences.